apps/modernization-ui/src/apps/search/patient/PatientCriteria/address.module.scss

```diff
@@ -0,0 +1,3 @@
+.address {
+    padding-bottom: 2rem;
```

Comment:
question (non-blocking):  This is being applied in on all of the criteria components, should we introduce a component to encapsulate this?  It could include those `label`, `select`, and `input` styles added to the accordion component to address issues of rendering those types of elements within it.

Reply:
If only all our components didn't have unpredictable margins and padding.

Reply:
it's really weird, how changing margins/padding work on details elements.